resources/views/home.twig

```diff
@@ -37,6 +37,9 @@
                 </div>
             </form>
         </section>
+        <div class="drop-overlay">
```

Comment:
I think that this could be appended via JS, because without JS it does not make any sense anyway